cloudbuild-k8s-generator.py

```diff
@@ -42,6 +42,9 @@
 timeout: 1200s # 20m
 options:
   machineType: 'N1_HIGHCPU_8'
+substitutions:
+  _CLUSTER_NAME: limani-integ
```

Comment:
The idea to use variables is to use them to hide internal names, rather than move them to default.

Default variable should have some generic value, not coupled with our infrastructure.

Reply:
This change allows running cloud build from CLI following https://github.com/GoogleCloudPlatform/click-to-deploy#manually-run-the-build